llvm/include/llvm/CodeGen/TargetLowering.h

```diff
@@ -2063,6 +2077,9 @@ class LLVM_ABI TargetLoweringBase {
 
   virtual bool isJumpTableRelative() const;
 
+  /// Retuen the minimum of largest number of comparisons in BitTest.
+  virtual unsigned getMinimumBitTestCmps() const;
```

Comment:
This doesn't need to be virtual if you have a member variable.

```suggestion
  unsigned getMinimumBitTestCmps() const;
```